src/libostree/ostree-repo-commit.c

```diff
@@ -680,7 +676,11 @@ _import_payload_link (OstreeRepo   *dest_repo,
   glnx_unref_object OtChecksumInstream *checksum_payload = NULL;
   g_autoptr(GFileInfo) file_info = NULL;
 
-  if (!_check_support_reflink (src_repo, dest_repo, &reflinks_supported, error))
+  /* The two repositories are on different devices */
+  if (src_repo->device != dest_repo->device)
+    return FALSE;
```

Comment:
This should be `return TRUE` right?

Reply:
yes, correct.  Fixed in a new fixup

Reply:
OK but that led me to consider "why weren't the tests failing", so I looked at that.  I ended up with the fixup :arrow_down: